domains/connor.json

```
@@ -1,11 +1,11 @@
 {
-  "description": "A website to show off my projects and or just blogs, etc etc.",
-  "repo": "https://github.com/ZeroTwoDevs/connor.github.io",
+  "description": "Personal portfolio.",
+  "repo": "https://github.com/ConnorDoesDev/connordoesdev.github.io",
   "owner": {
     "username": "Connor",
```

```suggestion
    "username": "ConnorDoesDev",
```

I don't know how the original PR was approved with this in the username field lol

---

No clue lol 
Sorry for that